rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+---
+feature: community-team
+start-date: 2021-07-31
+author: Irenes (Irene Knapp)
+co-authors: ashkitten
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Establish an official community team to make moderation decisions, model social norms, and mediate interpersonal problems.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not sustainable to have to call on Graham any time there's a spammer or conflict that requires moderation, and we'd like to help the community become more self-regulating.
+
+# Goals
+[goals]: #goals
+
+* Model and enforce social norms
+* Mediate disputes when doing so is beneficial to the community
+* Respect the privacy of community members
+* Foster newcomers and develop their leadership potential
+* Moderate all places where community members interact, including Discourse, Matrix, and GitHub
+* Ensure that collaboration on shared technical goals is possible
+* Ensure that the Nix community is a welcoming space for people from marginalized backgrounds
```

Comment:
I understand you to be saying that the problem is that we can't ensure that *all* people from marginalized backgrounds can be included. This is true, in the same sense that the entire RFC is aspirational and sets forth an ideal.

The line you're commenting on is in the "goals" section, which is informative rather than normative, meaning that it does not create rules that will be enforced going forward. The purpose of the goals section is to guide discussion on this thread. It makes sense to list an ideal as a goal, since the goals are the rubric by which we evaluate how well the rest of the RFC is doing its job.

Of course people from marginalized backgrounds are expected to follow the values, just as everyone else is. There's no exemption. The "accessibility" value, in your example, primarily means that the community team would try to help the person understand the values.